pkg/proxy/endpoint.go

```diff
@@ -23,6 +23,8 @@ type EndpointConfig struct {
 
 	Log *log.Logger
 
+	Timeout int64
```

Comment:
`Timeout` already exists in `HttpClient`

Reply:
👍 removed!